content/en/user-guide/aws/security-token-service/index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Security Token Service"
+linkTitle: "Security Token Service"
+description: Get started with Security Token Service on LocalStack
+---
+
+## Introduction
+
+Security Token Service (STS) is a service provided by Amazon Web Services (AWS) that enables you to grant temporary, limited-privilege credentials to users and applications. STS implements fine-grained access control and reduce the exposure of your long-term credentials. The temporary credentials, known as security tokens, can be used to access AWS services and resources based on the permissions specified in the associated policies.
+
+LocalStack supports STS via the Community offering, allowing you to use the STS APIs in your local environment to request security tokens, manage permissions, integrate with identity providers, and more. The supported APIs are available on our [API coverage page](https://docs.localstack.cloud/references/coverage/coverage_sts/), which provides information on the extent of STS's integration with LocalStack.
+
+## Getting started
+
+This guide is designed for users new to STS and assumes basic knowledge of the AWS CLI and our [`awslocal`](https://github.com/localstack/awscli-local) wrapper script.
+
+Start your LocalStack container using your preferred method. We will demonstrate how to assume an IAM Role, get caller identity and session token using the STS with the AWS CLI.
+
+### Create an IAM User
+
+You can create an IAM User and Role using the [`CreateUser`](https://docs.aws.amazon.com/STS/latest/APIReference/API_CreateUser.html) API. The IAM User will be used to assume the IAM Role. Run the following command to create an IAM User, named `localstack-user`:
+
+{{< command >}}
+$ awslocal iam create-user \
+    --user-name localstack-user
+{{< /command >}}
+
+You can generate access keys for the IAM user using the [`CreateAccessKey`](https://docs.aws.amazon.com/STS/latest/APIReference/API_CreateAccessKey.html) API. Run the following command to create an access key for the IAM user:
+
+{{< command >}}
+$ awslocal iam create-access-key \
+    --user-name localstack-user
+{{< /command >}}
+
+The following output would be retrieved:
+
+```bash
+{
+    "AccessKey": {
+        "UserName": "localstack-user",
+        "AccessKeyId": "ACCESS_KEY_ID",
+        "Status": "Active",
+        "SecretAccessKey": "SECRET_ACCESS_KEY",
+        "CreateDate": "2023-08-24T17:16:16Z"
+    }
+}
+```
+
```

Comment:
What exactly does this part have to do with STS? I assume it to be related with the `GetSessionToken` call, but that is not quite clear from the text.